recipe/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
 
   run:
     - python
-    - numpy >1.7.1
+    - numpy  >1.7.1<1.14  # [py27]
```

Comment:
The `# [py27]` comment applies to the whole line, so iirc you are restricting numpy to python 2 here.
I think you need a second line which has `not python2` for the un restricted numpy dep